br/pkg/lightning/importer/import.go

```diff
@@ -482,7 +490,7 @@ func NewImportControllerWithPauser(
 		preInfoGetter:       preInfoGetter,
 		precheckItemBuilder: preCheckBuilder,
 		encBuilder:          encodingBuilder,
-		tikvModeSwitcher:    local.NewTiKVModeSwitcher(tls, cfg.TiDB.PdAddr, log.FromContext(ctx).Logger),
+		tikvModeSwitcher:    local.NewTiKVModeSwitcher(tls, pdCli, log.FromContext(ctx).Logger),
```

Comment:
init another client for it? odd to have an object with close but not called

Reply:
I think creating two pd client is not effient and a bit confusing. pd has a background goroutine to update members.